woff2/index.html

```diff
@@ -913,69 +912,21 @@ <h3 id="woff20Header-0"><span class='secno'>3.2. </span>WOFF2 Header</h3>
       where data from various separate substreams are recombined to create a complete glyph
       record for each entry of the original <span class=tt>glyf</span> table.</p>
 
-      <div class="correction" id="p1">
```

Comment:
It looks like there's still some remaining proposed correction markup that needs to be removed for the overlap bitmap (line 1064)

Reply:
Thanks, fixed!